test/plugins/collections.test.js

```diff
@@ -289,4 +290,98 @@ describe('collection plugin test', () => {
             });
         });
     });
+
+    describe('PUT /collections/{id}', () => {
+        const id = testCollection.id;
+        let options;
+        let updatedCollectionMock;
+
+        beforeEach(() => {
+            options = {
+                method: 'PUT',
+                url: `/collections/${id}`,
+                payload: {
+                    name: 'updated name',
+                    description: 'updated description',
+                    pipelineIds: [123, 124]
+                },
+                credentials: {
+                    username,
+                    scope: ['user']
+                }
+            };
+
+            updatedCollectionMock = hoek.clone(collectionMock);
```

Comment:
The value inside the updatedCollectionMock does not matter actually. Only what returns from toJson matters